app/api.js

```diff
@@ -559,7 +559,9 @@ export const updateCompany = (companyId, companyFields) => {
           k =>
             k +
             ': ' +
-            (companyFields[k] ? '"' + companyFields[k] + '"\n' : null + '\n')
+            (companyFields[k]
+              ? '"' + companyFields[k].replace(/"/g, '\\"') + '"\n'
+              : null + '\n')
```

Comment:
Had to look up what this did. A comment might be useful here. Or maybe I'm just not with the JavaScript lingo yet.